README.md

```diff
@@ -103,7 +105,9 @@ module.exports = {
     ]
   }
   htmlLoader: {
-  	ignoreCustomFragments: [/\{\{.*?}}/]
+  	ignoreCustomFragments: [/\{\{.*?}}/],
+  	root: path.resolve(__dirname, 'assets'),
+  	attrs: ['img:src', 'link:href']
```

Comment:
Oops, sorry! Sure, I'll do it asap, thanks!
El 26 mar. 2016 17:29, "Tobias Koppers" <notifications@github.com> escribió:

> In README.md
> <https://github.com/webpack/html-loader/pull/54#discussion_r57515757>:
>
> > @@ -103,7 +105,9 @@ module.exports = {
> >      ]
> >    }
> >    htmlLoader: {
> > -  	ignoreCustomFragments: [/\{\{.*?}}/]
> > +  	ignoreCustomFragments: [/\{\{.*?}}/],
> > +  	root: path.resolve(__dirname, 'assets'),
> > +  	attrs: ['img:src', 'link:href']
>
> Could you correct the spacing in this section when touching it? It is
> mixed tabs and spaces. Should be only spaces here.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/webpack/html-loader/pull/54/files/0771e24f5f70dc68bdc765009af4808fe324d26b#r57515757>
>